src/utils/TimeSlots.js

```diff
@@ -130,7 +130,10 @@ export function getSlotMetrics({ min: start, max: end, step, timeslots }) {
 
       const rangeStartMin = positionFromDate(rangeStart)
       const rangeEndMin = positionFromDate(rangeEnd)
-      const top = (rangeStartMin / (step * numSlots)) * 100
+      const top =
+        rangeStartMin === rangeEndMin
+          ? (rangeStartMin / (step * numSlots)) * 98
```

Comment:
what's going on here with `98`, is there a deterministic reason to use 98%? This feels like a number that would only work with the default slot numbers.

Reply:
Yes you are correct after further testing with smaller step values it does make the events not sit in the correct space. I have modified the code for this section to factor in the step value so as to allow this to now work correct for any step value.